### DISCUSSION TITLE
Using Nuxt Module

### DISCUSSION THREAD

**Discussion by chriszoch** [scalar/scalar#1433] [Q&A]
Hello!

I have been trying to fool with this all day. I can't seem to figure it out without having to modify the code a lot in the package I downloaded "@scalar/nuxt". Anyways, I have been trying to use the module all day. Not matter what I try to do I just keep getting the same message over and over again. 

` ERROR  Error while requiring module @scalar/nuxt: Error: Cannot find module '/home/amritk/apps/scalar/scalar/node_modules/.pnpm/jiti@1.21.0/node_modules/jiti/lib/index.js'
Require stack:
- /**/node_modules/@scalar/nuxt/dist/module.mjs

[7:03:17 PM]  ERROR  Cannot restart nuxt:  Cannot find module '/home/amritk/apps/scalar/scalar/node_modules/.pnpm/jiti@1.21.0/node_modules/jiti/lib/index.js'
Require stack:
- /**/node_modules/@scalar/nuxt/dist/module.mjs
`

I have blocked out my directory because it's not nessisary. If I could get some help with this issue that would be amazing!

**Comment by amritk** [comment]
Hey @chriszoch we actually ran into some packaging issues. Sorry, I had deprecated the package for now. We are trying to sort these out asap so we can get the integration going!

![image](https://github.com/scalar/scalar/assets/2039539/668f3910-386a-4c74-aec8-edc412274789)

**Comment by amritk** [comment]
hey @chriszoch you can try the latest version 0.2.74 it should be working again!

**Comment by chriszoch** [comment]
I will check it later. Thank you so much.

Thank you,

Christopher Zoch
Business Development Lead

O: 540.994.9899 x130 <tel:5409949899;130>
D: 540.509.5727 <tel:5405095727>
C: 540.629.3636 <tel:5406293636>
***@***.*** ***@***.***>

KTI LTD
301 Madison Ave N
Pulaski VA, 24301 <https://maps.app.goo.gl/kYFkJx6msphgyJ3X7>
www.ktiltd.com <https://www.ktiltd.com/>
